### PR TITLE
Show ckey on roundend stats preference

### PR DESCRIPTION
**Pull request by BobJoga**
## About The Pull Request

Title. Basically a preference toggle indicating whether or not you want your ckey to be shown on roundend. Having this toggle disabled means your ckey will be spoofed with a random fake ckey.

image

**update:** to avoid confusion, the spoofed ckey will now show a _(ckey hidden)_ footnote on it's left.

## Why It's Good For The Game

nooooo i dont want people to know my character noooooo

## Changelog
:cl:
add: Showing ckey on roundend stats is now a preference toggle.
/:cl:

### CODE REVIEW

File: strings/spoofckeys.txt [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+bigchungus
+keanureeves
+ronaldmcdonald
+potionseller
+bigbillhells
+jonesbbqandfootmassage
+bigmantyrone
+bennyfromfalloutnewvegas
+crayoneater
+exadv1
```
Comment (Afya212) [inline strings/spoofckeys.txt:22]:
```suggestion
exadv1
```

**Comment by Afya212** [comment]
Wouldn't it be nicer to just get ride of the "[ckey] was" part? Instead of putting a random ckey.

**Comment by Afya212** [comment]
Wouldn't it be nicer to just get rid of the "[ckey] was" part? Instead of putting a random ckey.

**Comment by BobJoga** [comment]
> Wouldn't it be nicer to just get rid of the "[ckey] was" part? Instead of putting a random ckey.

i find the latter funnier

**Comment by BobJoga** [comment]
> > Wouldn't it be nicer to just get rid of the "[ckey] was" part? Instead of putting a random ckey.
> 
> i find the latter funnier

image

**Comment by BobJoga** [comment]
Rohesie has devised a better version of this.

**Pull request closed without merging**